Tests/AttributedStringStyleTests.swift

```diff
@@ -117,6 +117,36 @@ class StringStyleTests: XCTestCase {
         }
     }
 
+    func testPronuciationStyle() {
+        if #available(iOS 11, tvOS 11, watchOS 4, *) {
```

Comment:
Wonder if there should be a convenience for this `#available` set since using it all over.

Reply:
Good point. I forgot I could just wrap this in a function/closure. I'll check it out and see where it makes sense to add it.

Reply:
Doesn't look like there's a way to do this that increases clarity, and it obfuscates the intent a little bit.